.travis.yml

```diff
@@ -12,6 +12,11 @@ matrix:
       env: PYTHON_VERSION=python3 ONNX_ML=0
       language: python
       python: "3.6"
+    - os: linux
+      sudo: required
+      env: PYTHON_VERSION=python3 ONNX_ML=0 DEBUG_MODE=1
```

Comment:
Would it be better to directly use `DEBUG=1` (instead of `DEBUG_MODE`) here?

Reply:
`DEBUG` seems a very common environment variable... Maybe `ONNX_DEBUG`?

Reply:
My logic was along the lines of @wschin's comment -- I've renamed it to `ONNX_DEBUG`.